gui/wxpython/vdigit/toolbars.py

```diff
@@ -14,15 +14,23 @@
 @author Martin Landa <landa.martin gmail.com>
 @author Stepan Turek <stepan.turek seznam.cz> (handlers support)
 """
+
+from ctypes import c_char_p, pointer
+
 import wx
 
 from grass import script as grass
+from grass.lib.vector import GV_BOUNDARY, GV_LINE, GV_LINES, Map_info, \
+    Vect_close, Vect_get_line_type, Vect_get_num_lines, Vect_line_alive, \
+    Vect_line_length, Vect_new_cats_struct, Vect_new_line_struct, \
+    Vect_open_old, Vect_read_line
```

Comment:
The use of ctypes and the generated ctypes interface to libraries is tricky already at the import level. Other places in GUI which do this are (or should be) very careful about import. It is best to completely avoid this in GUI and solve it in a separate module (in Python or C).